tests/server/ckeditor.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import React from 'react';
+import ReactDOMServer from 'react-dom/server';
+import { expect } from 'chai';
+
+import CKEditorDev from '../../src/ckeditor.jsx';
+import CKEditorBuilt from '../../dist/ckeditor.js';
+
+describe( 'CKEditor Component SSR', () => {
+	describe( 'basic rendering', () => {
+		createTest( 'returns appropriate HTML', CKEditor => {
```

Comment:
Hmm, won't we be then testing external code instead of ours? I don't see much sense with such test as rendering the whole app to the string is basically gluing strings returned from all components. If our component returns something sensible, then it should work flawlessly.